apps/nextjs-website/react-components/components/Redirect/Redirect.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+
+import { useRouter } from 'next/navigation';
+import { useEffect, useRef } from 'react';
+import type { RedirectProps } from '../../types/Redirect/Redirect.types';
+
+const Redirect = ({ redirectCode, redirectURL }: RedirectProps) => {
+    const router = useRouter();
+    const hasRedirectedRef = useRef(false);
+
+    useEffect(() => {
+        const destination = redirectURL?.trim();
+
+        if (!destination || hasRedirectedRef.current) {
+            return;
+        }
+
+        hasRedirectedRef.current = true;
+
+        if (redirectCode === '301') {
+            router.replace(destination);
+            return;
+        }
+
+        router.push(destination);
+    }, [redirectCode, redirectURL, router]);
+
+    return null;
+};
```

Comment:
```suggestion
const Redirect = ({ redirectURL }: RedirectProps) => (
  <>
    <meta httpEquiv='refresh' content={`0; url=${redirectURL}`} />
    <p>
      Redirecting to <a href={redirectURL}>{redirectURL}</a>...
    </p>
  </>
);
```
Simplify using meta http-equiv tag + Add some feedback letting both the user and crawlers know that the page is being redirected.

(The text has to be minimally styled and localized)